addons/account_edi_peppol/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': 'PEPPOL support for Accounting',
+    'version': '1.0',
+    'category': 'Accounting/Accounting',
```

Comment:
no description ? :sob: 

edit: unfortunately, that remark applies for account_edi_ubl and account_edi_facturex. I'm expecting to be able, at least, to find there the supported version(s) of the edi.format.